one/src/main.rs

```diff
@@ -287,11 +317,14 @@ impl Daemon {
             .build(sql_pool.clone())
             .await?;
 
+        let bind_address = opts.bind_address.clone();
+        let metrics_bind_address = opts.metrics_bind_address.clone();
```

Comment:
What is this change for? Why do we clone the address?

Reply:
Removed

This was a change from a different branch by accident.
 I was changing the way we pass the opts to have all DaemonOpts stored on the Daemon struct.
`daemon.opts` so they could be used in `daemon.run()` not just `daemon.build()`.